plugin/valgrind.vim

```diff
@@ -365,31 +363,31 @@ function! s:Show_Help( first_time )
     setl modifiable
 
     if !a:first_time
-    normal G$
-    if ( search( '^$', 'w' ) > 0 )
-        normal d1G
-    endif
+        normal! G$
+        if ( search( '^$', 'w' ) > 0 )
+            normal! d1G
+        endif
     endif
 
     if exists("s:show_help") && s:show_help == 1
-    call append(0, '" <enter> : Jump to error')
-    call append(1, '" o : Jump to error in new window')
-    call append(2, '" <space> : Show error')
-    call append(3, '" x : Zoom-out/Zoom-in valgrind window')
-    call append(4, '" + : Open a fold')
-    call append(5, '" - : Close a fold')
-    call append(6, '" * : Open all folds')
-    call append(7, '" q : Close the valgrind window')
-    call append(8, '" ? : Remove help text')
-    call append(9, '')
+        call append(0, '" <enter> : Jump to error')
+        call append(1, '" o : Jump to error in new window')
+        call append(2, '" <space> : Show error')
+        call append(3, '" x : Zoom-out/Zoom-in valgrind window')
+        call append(4, '" + : Open a fold')
+        call append(5, '" - : Close a fold')
+        call append(6, '" * : Open all folds')
+        call append(7, '" q : Close the valgrind window')
+        call append(8, '" ? : Remove help text')
+        call append(9, '')
         let s:show_help = 0
     else
-    call append(0, '" Press ? to display help text')
-    call append(1, '')
+        call append(0, '" Press ? to display help text')
+        call append(1, '')
         let s:show_help = 1
     endif
 
-    normal 1G
+    normal! 1G
```

Comment:
This is a valuable bug fix, but I would argue it is more than "Fixing indentation". Could you update the commit message and then I'll pull it in.